webdriver/tests/support/fixtures.py

```diff
@@ -18,16 +18,14 @@
 
 
 _current_session = None
+_current_bidi_session = None
```

Comment:
I'm worried about having more than one global session. I think it would make sense to have a `bidi_session` fixture but for it to actually share the same session as the `_current_session`; if the bidi status changes we'd need to restart the session just like we do with the capabilities.